src/expensive_services_detail/cost_metrics_of_expensive_services.py

```diff
@@ -118,7 +142,7 @@ def lambda_handler(event, context):
     # Loop through each region
     for region in regions:
         try:
-            # print(region)
+            # region_name = region_names.get(region, "unknown region name")
```

Comment:
please remove comments

Reply:
Added in d0ff7d06072eb4c290cb31d2d612c49dd5c6e6ed